app/controllers/orders/detail.js

```diff
@@ -159,6 +133,18 @@ export default GoodcityController.extend({
   },
 
   actions: {
+    loadOrdersPackages(pageNo) {
+      const params = this.trimQuery(
+        _.merge(
+          { order_id: this.get("orderId") },
+          this.getPaginationQuery(pageNo)
+        )
+      );
+      return this.get("store")
+        .query("orders_package", params)
+        .then(results => results);
```

Comment:
this `then` statement doesn't seem needed since you're just returning the value straight away